hack/coverage.sh

```diff
@@ -13,9 +13,28 @@ declare -a EXCLUDE_FILES_FROM_COVERAGE=("nodemaintenance_controller_init.go")
 # delete coverage files (if present)
 find . -name ${COVERAGE_FILE} | xargs rm -f
 
-# run ginkgo with coverage result line
-${GINKGO} ${GINKGO_ARGS} ./pkg/ ./cmd/ | sed  '/coverage:.*$/d'
-GSTAT=${PIPESTATUS[0]}
+while [ true ]; do
+
+		# run ginkgo with coverage result line
+		${GINKGO} ${GINKGO_ARGS} ./pkg/ ./cmd/ 2>&1 | sed  '/coverage:.*$/d'  | tee msg
```

Comment:
If ginkgo is failing for an unknown reason, maybe it makes sense to stop erasing parts of it's output incase it holds some clue to the failures.

In any case, can we remove this change and continue the conversation in the other PR you created to address this? 
   https://github.com/kubevirt/node-maintenance-operator/pull/78/files/c0c1a0bd5bcddd27543089dbf198adea940a76c3#r424158734

Reply:
removed